### ISSUE TITLE
The ST0102 Entries are Commented Out of the Code

### ISSUE THREAD

**Issue by lerman06470** [paretech/klvdata#33]
That corresponds to tag 48 in ST0601. I've tried un-commenting that section, but the code still doesn't seem to display that tag.

Tag 94 (which is part of the ST0902 minimal data) set is also commented out, as well as tag 95.

**Comment by All4Gis** [comment]
Yes, Any Pull request is welcome, maybe we need make more changes in the code not only un-comment this tags
